README.md

```diff
@@ -133,4 +133,20 @@ nvm install 22
 node -v # should print `v22.11.0`
 # verifies the right npm version is in the environment
 npm -v # should print `10.9.0`
+
 ```
```

Comment:
@CharlieJCJ added this section here - what do you think now? (it is collapsible so that only developers see this and does not add to normal users of the package who come to git to get a quick start guide.

Reply:
Content is good, for the markdown formatting, I changed a little bit after looking at a few other repos' README. Two nits: 1) we can make the subsection foldable, so user can see an outline of what is under Development. 2) collapse on h4, instead of h2 for arrow aligned with center of the font. 

Try the following

## Development

<details>
<summary><h4 style="display: inline-block">Environment Setup with <code>.env</code> file (optional)</h4></summary>

Copy `.env.example` to create your `.env` file:
```bash
cp .env.example .env
```

Available environment variables:
- `OPENAI_API_KEY`: (Required) Your OpenAI API key for language model requests
- `CURATOR_CACHE_DIR`: (Optional) Custom cache directory, defaults to `~/.cache/curator`. Used to store datasets, model responses, and metadata for caching and viewer functionality.

</details>